iam.tf

```diff
@@ -36,3 +53,11 @@ resource "aws_iam_role" "sns_feedback_role" {
     var.sns_topic_feedback_role_tags,
   )
 }
+
+resource "aws_iam_role_policy" "sns_feedback_role" {
```

Comment:
create a policy and attach it instead of inline policy. name should not be hardcoded  - you can see other modules for references

Reply:
Ok, can do! Can you point me to a reference? (I looked at this repo, vpc, rds, and s3, and I don't see what you are referring to)